src/Model/GServer.php

```diff
@@ -87,9 +87,11 @@ class GServer
 
 	// Standardized endpoints
 	const DETECT_STATISTICS_JSON = 100;
-	const DETECT_NODEINFO_1 = 101;
-	const DETECT_NODEINFO_2 = 102;
-	const DETECT_NODEINFO_210 = 103;
+	const DETECT_NODEINFO_1 = 101; // Nodeinfo Version 1.0
+	const DETECT_NODEINFO_20 = 102; // Nodeinfo Version 2.0
+	const DETECT_NODEINFO_210 = 103; // Nodeinfo 2 Version 1.0
+	const DETECT_NODEINFO_21 = 104; // Nodeinfo Version 2.1
+	const DETECT_NODEINFO_22 = 105; // Nodeinfo Version 2.2
```

Comment:
Renaming suggestion to thwart potential and likely confusion. We probably should do the same for the class names.
```suggestion
	const DETECT_NODEINFO1_10 = 101; // Nodeinfo (1) Version 1.0
	const DETECT_NODEINFO1_20 = 102; // Nodeinfo (1) Version 2.0
	const DETECT_NODEINFO2_10 = 103; // Nodeinfo 2 Version 1.0
	const DETECT_NODEINFO2_21 = 104; // Nodeinfo 2 Version 2.1
	const DETECT_NODEINFO2_22 = 105; // Nodeinfo 2 Version 2.2
```

Reply:
[Nodeinfo2](https://github.com/jaywink/nodeinfo2) is a fork of [Nodeinfo](https://github.com/jhass/nodeinfo). It is not version 1 or version 2.